dashboard/app/views/weblab_host/index.html.haml

```diff
@@ -1,6 +1,8 @@
 %html{spellcheck: 'false'}
   %head
+    = javascript_include_tag 'application'
```

Comment:
The `brambleHost.js` code runs in an iframe hosted by this page. Since I'm now trying to download the assets in this iframe, using the same code as we do elsewhere in code-studio, this frame now requires jquery. Rather than just grab jquery by itself, I grab the exact the same jquery bundled via the sprockets/rails asset pipeline that we serve from all other dashboard pages by adding this. (It doesn't happen automatically, because this controller doesn't inherit from `ApplicationController`).